solutions/project/hub-env/fortigate/service-account.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Service Account
+## GCP SA Fortigate SDN read-only
+## AC-2(A) - One GCP service account is created and attached to both Fortigates VM.
+## AC-3(7) - Creates lower priv service account to attach to Fortigate appliances.
```

Comment:
For the controls do we want to list them as a metadata annotation or label so that they are more easily discovered in cluster and/or the GCP UI?

Reply:
IMO, it can be addressed later as we don't have any standard defined yet for labels or annotations for identifying security controls. I will reference this conversation in a new issue

Reply:
issue #301 created